apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,339 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Bigtable.V2.Snippets
+{
+    public class BigtableClientSnippets
+    {
+        public void CheckAndMutateRow()
+        {
+            // Snippet: CheckAndMutateRow(TableName,RowKey,RowFilter,IEnumerable<Mutation>,IEnumerable<Mutation>,CallSettings)
+            // Create client
+            BigtableClient bigtableClient = BigtableClient.Create();
+            // Initialize request argument(s)
+            TableName tableName = new TableName("[PROJECT]", "[INSTANCE]", "[TABLE]");
+            RowKey rowKey = "r1";
+            RowFilter predicateFilter = RowFilters.Chain(
+                RowFilters.FamilyNameRegex(@"cf\d"),
+                RowFilters.ColumnQualifierRegex(@"[a-z]\d"),
+                RowFilters.ValueRegex(@"test-value"));
+            IEnumerable<Mutation> trueMutations = new Mutation[]
```

Comment:
Do you just mean for the snippet code, or do you mean the parameter names for the method should change as well? They are documented [here](https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#google.bigtable.v2.CheckAndMutateRowRequest) as true_mutations and false_mutations. Also, these are the parameter names for the auto-generated CheckAndMutateRow method. So while I think your names are a bit clearer, for consistency it might make sense to stick with these.